packages/@aws-cdk/aws-glue-alpha/test/job.test.ts

```diff
@@ -796,6 +808,21 @@ describe('Job', () => {
       });
     });
 
+    describe('enabling job run queuing', () => {
+      beforeEach(() => {
+        job = new glue.Job(stack, 'Job', {
+          ...defaultProps,
+          jobRunQueuingEnabled: true,
+        });
+      });
+
+      test('should set JobRunQueuingEnabled', () => {
+        Template.fromStack(stack).hasResourceProperties('AWS::Glue::Job', {
+          JobRunQueuingEnabled: true,
+        });
+      });
+    });
```

Comment:
This description may be sufficient.

```suggestion
    test('enabling job run queuing', () => {
        job = new glue.Job(stack, 'Job', {
          ...defaultProps,
          jobRunQueuingEnabled: true,
        });
        Template.fromStack(stack).hasResourceProperties('AWS::Glue::Job', {
          JobRunQueuingEnabled: true,
        });
    });
```